scripts/waf-sam/os-fire-sam/README

```diff
@@ -0,0 +1,64 @@
+## Project Overview
+
+This directory contains an AWS SAM (Serverless Application Model) project for deploying an AWS WAF dashboard on Amazon OpenSearch Service. It includes:
+
+- **template.yaml**: The SAM/CloudFormation template defining all AWS resources (Lambda functions, IAM roles, custom resources, etc.).
+- **Python handlers** for custom resources and processing:
+  - `kibana_customizer.py` – loads index templates and OpenSearch Dashboards saved objects (visualizations and dashboards).
+  - `processor_function.py` – core log-processing Lambda logic (Firehose processing).
+  - `es_cognito_auth_setter.py` – sets up Cognito authentication for the OpenSearch domain.
+  - `user_pool_domain_setter.py` – custom resource to configure a Cognito User Pool domain.
+- **custom/** directory: JSON definitions for OpenSearch index templates and OpenSearch Dashboards saved objects:
+  - `template.json` – Elasticsearch index template for AWS WAF logs.
+  - `.json` files for each visualization (e.g., `top10countries.json`, `allvsblockedrequests.json`, etc.) and the dashboard (`dashboard.json`).
+- **cfnresponse/**: AWS CloudFormation response helper module.
+- **urllib3/**: Bundled HTTP library dependency for Lambda functions.
+
+## Directory Structure
+
+```
+├── template.yaml
+├── kibana_customizer.py
+├── processor_function.py
+├── es_cognito_auth_setter.py
+├── user_pool_domain_setter.py
+├── custom/
+│   ├── template.json
+│   ├── allvsblockedrequests.json
+│   ├── allcountries.json
+│   ├── blockedcountries.json
+│   ├── dashboard.json
+│   ├── executedrules.json
+│   ├── filters.json
+│   ├── httpmethods.json
+│   ├── httpversions.json
+│   ├── numberofallrequests.json
+│   ├── numberofblockedrequests.json
+│   ├── requestcount.json
+│   ├── top10countries.json
+│   ├── top10hosts.json
+│   ├── top10ip.json
+│   ├── top10rules.json
+│   ├── top10uris.json
+│   ├── top10useragents.json
+│   ├── top10webacl.json
+│   └── uniqueipcount.json
+├── cfnresponse/
+│   └── __init__.py
+└── urllib3/
+    └── ...
+```
+
+## Usage
+
+1. **Build & Deploy**:
+   ```bash
```

Comment:
should we add statement to export aws credentials or is that obvious for any template?

Reply:
added